sdk/python/tests/unit/infra/scaffolding/test_repo_config.py

```diff
@@ -247,7 +246,32 @@ def test_auth_config():
         expect_error="invalid authentication type=not_valid_auth_type",
     )
 
-    oidc_repo_config = _test_config(
+    oidc_server_repo_config = _test_config(
+        dedent(
+            """
+        project: foo
+        auth:
+            type: oidc
+            client_id: test_client_id
+            auth_discovery_url: http://localhost:8080/realms/master/.well-known/openid-configuration
```

Comment:
for the sake of consistency, `master` should be replaced by `test_client_id` in the URL